nipoppy/workflows/pipeline_store/zenodo.py

```diff
@@ -81,7 +81,8 @@ def run_main(self):
         """Run the main workflow."""
         if not self.assume_yes:
             continue_ = Confirm.ask(
-                "The Nipoppy pipeline will be uploaded/updated on Zenodo,"
+                "The Nipoppy pipeline will be uploaded/updated on Zenodo"
+                f"{' (sanbox) 'if self.zenodo_api.sandbox else ""},"
```

Comment:
This is causing tests to fail so I will go ahead and fix it

```suggestion
                f"{' (sanbox) 'if self.zenodo_api.sandbox else ''},"
```